shared/src/iosTest/kotlin/com/twilio/passkeys/mocks/PayloadMocks.kt

```diff
@@ -0,0 +1,248 @@
+package com.twilio.passkeys.mocks
+
+import com.twilio.passkeys.models.AuthenticatePasskeyRequest
+import com.twilio.passkeys.models.AuthenticatePasskeyRequestPublicKey
+import com.twilio.passkeys.models.AuthenticatePasskeyResponse
+import com.twilio.passkeys.models.AuthenticatorSelection
+import com.twilio.passkeys.models.CreatePasskeyRequest
+import com.twilio.passkeys.models.CreatePasskeyResponse
+import com.twilio.passkeys.models.KeyCredential
+import com.twilio.passkeys.models.PubKeyCredParams
+import com.twilio.passkeys.models.Rp
+import com.twilio.passkeys.models.User
+
+val createPasskeyChallengePayload =
+  """
+  {
+    "rp": {
+      "id": "example.com",
+      "name": "Example"
+    },
+    "user": {
+      "id": "PLACEHOLDERIDWUU0ZmQzYWFmNGU0NTMyNGQwZjNlMTM0N",
+      "name": "user1",
+      "displayName": "User One"
+    },
+    "challenge": "WUYwNDhkMWE3ZWMzYTJhNjk3MDA1OWMyNzY2YmJjN2UwZg",
+    "pubKeyCredParams": [
+      {
+        "type": "public-key",
+        "alg": -7
+      }
+    ],
+    "timeout": 600000,
+    "excludeCredentials": [
+
+    ],
+    "authenticatorSelection": {
+      "authenticatorAttachment": "platform",
+      "requireResidentKey": false,
+      "residentKey": "preferred",
+      "userVerification": "preferred"
+    },
+    "attestation": "none"
+  }
+  """.trimIndent()
+
+const val RP_ID = "example.com"
+const val RP_NAME = "Example"
+const val USER_ID = "PLACEHOLDERIDWUU0ZmQzYWFmNGU0NTMyNGQwZjNlMTM0N"
+const val USER_NAME = "user1"
+const val USER_DISPLAY_NAME = "User One"
+const val CREATE_CHALLENGE = "WUYwNDhkMWE3ZWMzYTJhNjk3MDA1OWMyNzY2YmJjN2UwZg"
+const val PUB_KEY_CRED_TYPE = "public-key"
+const val PUB_KEY_CRED_ALG = -7
+const val TIMEOUT = 600000L
+val keyCredential =
+  KeyCredential(
+    id = "6ySmhJd6qGUMCthiqszyb4Od4U6TFn0v3DLz-1EZrNQ",
```

Comment:
This can be flagged as a secret

Reply:
Same for values below `RAW_ID`, `ATTESTATION_OBJECT`, `AUTHENTICATE_CHALLENGE`, `CLIENT_DATA_JSON_CREATE`, `SIGNATURE`, `USER_HANDLE`. If we can use placeholder without letters will be better, if we need the values encoded or following a format, we should use dummy values so they are not considered secrets